pycon/finaid/urls.py

```diff
@@ -14,4 +14,6 @@
     url(r"^mail/(?P<pks>[0-9,]+)/$", finaid_email, name="finaid_email"),
     url(r"^message/(?P<pks>[0-9,]+)/$", finaid_message, name="finaid_message"),
     url(r"^download/$", finaid_download_csv, name="finaid_download_csv"),
+    url(r"^receipt_form/$", receipt_form, name="receipt_form"),
+    url(r"^receipt_delete/(?P<pk>\d+)/$", receipt_delete , name="receipt_delete")
```

Comment:
Let's ask whether they want to let users delete their receipts. They very well might want to, but they didn't explicitly ask for that feature.